### ISSUE TITLE
amp-selector clear action missing in Action documentation

### ISSUE THREAD

**Issue by caroqliu** [ampproject/amphtml#27699]
The `clear` action described on `amp-selector` [documentation page](https://amp.dev/documentation/components/amp-selector/#clearing-selections) should also be listed among its actions in the [AMP Actions and Events documentation](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/?referrer=ampproject.org#amp-selector_1).
